entrypoint.sh

```diff
@@ -21,16 +21,36 @@ URI=https://api.github.com
 API_HEADER="Accept: application/vnd.github.v3+json"
 AUTH_HEADER="Authorization: token $GITHUB_TOKEN"
 
-pr_resp=$(curl -X GET -s -H "${AUTH_HEADER}" -H "${API_HEADER}" \
-          "${URI}/repos/$GITHUB_REPOSITORY/pulls/$PR_NUMBER")
+MAX_RETRIES=${MAX_RETRIES:-6}
+RETRY_INTERVAL=${RETRY_INTERVAL:-10}
+REBASEABLE=""
+pr_resp=""
+for ((i = 0 ; i < $MAX_RETRIES ; i++)); do
+	pr_resp=$(curl -X GET -s -H "${AUTH_HEADER}" -H "${API_HEADER}" \
+		"${URI}/repos/$GITHUB_REPOSITORY/pulls/$PR_NUMBER")
+	REBASEABLE=$(echo "$pr_resp" | jq -r .rebaseable)
+	if [[ "$REBASEABLE" == "null" ]]; then
+		echo "The PR is not ready to rebase, retry after $RETRY_INTERVAL seconds"
+		sleep $RETRY_INTERVAL
+		continue
+	else
+		break
+	fi
+done
+
+if [[ "$REBASEABLE" != "true" || "$i" == "$MAX_RETRIES" ]] ; then
+	echo "GitHub doesn't think that the PR is rebaseable!"
+	echo "API response: $pr_resp"
```

Comment:
Let's not print the response here. It was a CVE https://bugs.chromium.org/p/project-zero/issues/detail?id=2070

Reply:
https://github.com/cirrus-actions/rebase/pull/75/commits/e4987e76cfabed2bbe500c3b855a435b58cf7947